runtimes/deno-runtime/Dockerfile

```diff
@@ -6,6 +6,8 @@ LABEL org.opencontainers.image.source="https://github.com/keptn/lifecycle-toolki
     org.opencontainers.image.vendor="Keptn" \
     org.opencontainers.image.licenses="Apache-2.0"
 
+RUN apk add --no-cache libcrypto3~=3.0.12 libssl3~=3.0.12
+
```

Comment:
too bad deno image has not yet bumped their alpine version :( there is a pr open since december...

Reply:
yeah...